docs/resources/scalr_vcs_provider.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: "scalr"
+page_title: "Scalr: scalr_vcs_provider"
+sidebar_current: "docs-resource-scalr-vcs-provider"
+description: |-
+  Manages vcs provider.
+---
+
+# scalr_vcs_provider Resource
+
+Manage the Scalr VCS provider. Create, update and destroy.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "scalr_vcs_provider" "example" {
+  name="example-github"
+  account_id = "acc-xxxxx"
+  vcs_type="github"
+  token="token"
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) Name of the vcs provider.
+* `vcs_type` (Required) The vcs provider type is one of `github`, `github_enterprice`, `gitlab`, `github_enterprice`, `bitbucket_enterprise`. 
+   The other providers currently is not supported in resource.
+* `token` (Required) The personal access token for provider
+  * Github token can be generated by url https://github.com/settings/tokens/new?description=example-vcs-resouce&scopes=repo
+  * Gitlab token can be generated by url https://gitlab.com/-/profile/personal_access_tokens?name=example-vcs-resouce&scopes=api,read_user,read_registry
```

Comment:
@penja Let's add the instructions for bitbucket as well

Reply:
🙈